plugin/entryAttributes.go

```diff
@@ -55,6 +85,8 @@ type EntryAttributes struct {
 	mtime   time.Time
 	ctime   time.Time
 	crtime  time.Time
+	os      OS
+	hasOS   bool
```

Comment:
Some things will not have any OS information because they're not a computer. I think it makes sense to reflect that directly, rather than having an empty struct.

Reply:
Also makes sense.